06 Rendering Lists and Conditional Content/01-starting-setup/src/Todo.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export default function Todo(props) {
+    return <li>{props.todoItems}</li>
```

Comment:
Implement a search functionality on the todoitems

The text search should have debounce feature, so that search gets initiated only after user stops typing

secondly, it should be a free search, basically i can search for anything in the todoItems.